scm_v3c/scm3c_hw_interface.c

```diff
@@ -1457,7 +1458,8 @@ void clear_asc_bit(unsigned int position) {
     scm3c_hw_interface_vars.ASC[index] &=
         ~(0x80000000 >> (position - (index << 5)));
 
-    // Needed for no 1.1V -> VDDD fix
+    // 1.1V FIX (Adds delay before trying to restore registers from stack)
+    // Without NOP, position variable is zero (when it shouldn't be)
```

Comment:
What do you mean by position variable?

Reply:
The variable being passed is named position. Well at least the local variable is called position. But the variable that's being set to zero is the original variable being passed to the function.